src/main/java/com/rapid7/container/analyzer/docker/packages/DotNetParser.java

```diff
@@ -51,7 +52,11 @@ public Set<Package> parse(File input, OperatingSystem operatingSystem) throws IO
           String name = getValueForAttribute(element, "id");
           String version = getValueForAttribute(element, "version");
           String description = getValueForAttribute(element, "description");
-          packages.add(new Package(source, PackageType.DOTNET, operatingSystem, name, version, description, null, null, null, null));
+          try {
+            packages.add(new Package(source, PackageType.DOTNET, operatingSystem, name, version, description, null, null, null, null));
+          } catch (PackageValidationException pve) {
+            LOGGER.info(pve.getMessage());
```

Comment:
Do you really want this at INFO level?

Reply:
It happens a lot is the thing. I thought about putting it at warn, but chances are the people using the app aren't going to change anything based off that warning. I'm kinda on the fence about it. What do you think?

Reply:
Like how often? I agree with your second point but it is a bit weird to cover up bad things with info logging